sdb/commands/threads.py

```diff
@@ -0,0 +1,72 @@
+#
+# Copyright 2019 Delphix
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# pylint: disable=missing-docstring
+
+import argparse
+from typing import Any, Dict, Iterable, List, Tuple
+from collections import defaultdict
+
+import drgn
+from drgn.helpers.linux.pid import for_each_task
+
+import sdb
+from sdb.commands.internal.table import Table
+from sdb.commands.stacks import Stacks
+
+class Threads(sdb.Locator, sdb.PrettyPrinter):
+    """
+    Print information about all or a single thread
+
+    task - address of the task_struct
+    state - the state of the thread
+    pid - the pid of the thread's process
+    prio - the priority of the thread
+    comm - the thread's command
```

Comment:
More of a future note but it would be nice to add an example here (could be the same as the one that we use for the test suite too)